aten/src/TH/THTensor.cpp

```diff
@@ -54,3 +56,69 @@ void THTensor_free(THTensor *self)
     }
   }
 }
+
+// On a high level,
+// 1. separate tensor->size into chunks of dimensions, where the dimensions are
+//    ``contiguous'' in each chunk, i.e., stride[i] = size[i+1] * stride[i+1]
+// 2. view_size must be able to be separated into same number of chunks as tensor->size was separated into,
```

Comment:
Is this comment still valid? No mention of `view_size`.

Reply:
You are correct, the two implementations used different variable names.